source/docs/software/advanced-controls/introduction/control-system-basics.rst

```diff
@@ -33,6 +44,15 @@ The figure below shows a system with a hypothetical input and output. Since the
 .. image:: images/control-system-basics-whatisgain.png
    :alt: A system diagram with hypothetical input and output
 
+What is a Model?
+----------------
+
+A *model* of your mechanism is a set of math equations which describe its expected behavior.
```

Comment:
I'd say "a mathematical description of its behavior."  You can go more in depth, and talk about how what this really means is some sort of law relating its *inputs* and *outputs.*